### ISSUE TITLE
Control over button markup

### ISSUE THREAD

**Issue by mhulse** [artoodetoo/formToWizard#3]
Thoughts on having the markup for button navigation be in HTML template or as a template string?

At a minimum, maybe have an option for the parent for the buttons to be appended into?

**Comment by artoodetoo** [comment]
I agree, it would be nice to improve this. I'll think about how to do it the most simple way :)
Thank you for your interest.
